web-ui/src/pages/AnniversaryReportPage.jsx

```diff
@@ -7,10 +7,10 @@ import Autocomplete from "@mui/material/Autocomplete";
 
 import "./BirthdayAnniversaryReportPage.css";
```

Comment:
This file was renamed to indicate that it is only used for anniversaries, not birthdays.